.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    docker:
+      - image: circleci/elixir:1.4-node-browsers
+        environment:
+          MIX_ENV: test
+          CLIENT_ID: fake
+          CLIENT_SECRET_ID: fake
+          OUTBOUND_EMAIL_DOMAIN: http://localhost
+          INBOUND_EMAIL_DOMAIN: http://localhost
+          SLACK_WEBHOOK_URL: https://fakeslack/webhook
+      - image: circleci/postgres:9.5
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: constable_api_test
+          POSTGRES_PASSWORD:
+
+    working_directory: ~/app
+
+    steps:
+      - checkout
+
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+
+      - restore_cache:
+          keys:
+            - v1-mix-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
+            - v1-mix-cache-{{ .Branch }}
+            - v1-mix-cache
+      - restore_cache:
+          keys:
+            - v1-build-cache-{{ .Branch }}
+            - v1-build-cache
+      - run: mix do deps.get, compile --warnings-as-errors
+      - save_cache:
+          key: v1-mix-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
+          paths: "deps"
+      - save_cache:
+          key: v1-mix-cache-{{ .Branch }}
+          paths: "deps"
+      - save_cache:
+          key: v1-mix-cache
+          paths: "deps"
+      - save_cache:
+          key: v1-build-cache-{{ .Branch }}
+          paths: "_build"
+      - save_cache:
+          key: v1-build-cache
+          paths: "_build"
+
+      - restore_cache:
+          keys:
+            - v1-npm-cache-{{ .Branch }}-{{ checksum "assets/package-lock.json" }}
+            - v1-npm-cache-{{ .Branch }}
+            - v1-npm-cache
+      - run:
+          working_directory: assets
+          command: npm install && node_modules/.bin/brunch build
+      - save_cache:
+          key: v1-npm-cache-{{ .Branch }}-{{ checksum "assets/package-lock.json" }}
+          paths: "assets/node_modules"
+      - save_cache:
+          key: v1-npm-cache-{{ .Branch }}
+          paths: "assets/node_modules"
+      - save_cache:
+          key: v1-npm-cache
+          paths: "assets/node_modules"
+
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+
+      - run: bin/test_suite
+
+      - store_test_results:
+          path: _build/test/junit
```

Comment:
what's this doing?

Reply:
I grabbed that from the [elixir language guide](https://circleci.com/docs/2.0/language-elixir/) from circle ci. I believe that's where we'd store test results for Test Summary if we were using more than one container. We probably don't need it for now since we're only using one.